lib/dor/services/client/marcxml.rb

```diff
@@ -8,24 +8,26 @@ class Marcxml < VersionedService
         # Gets MARCXML corresponding to a barcode or catkey
         # @param barcode [String] required string representing a barcode
         # @param catkey [String] required string representing a catkey
+        # @param folio_instance_hrid [String] required string representing a Folio instance HRID
```

Comment:
I see in [DSA's openapi.yml ](https://github.com/sul-dlss/dor-services-app/blob/04126203fe52f98615b9c95d3ff1cdebe0438554/openapi.yml#L79 )that these params are not required, and they have default values. Should the docs be updated here? Wanted to make sure I'm understanding this correctly.

Reply:
I don't **think** so?

Reply:
`#check_args` takes care of that, so I removed the word "required" from the docs since it is confusing to read that each one is required.